### DISCUSSION TITLE
github.com timeout

### DISCUSSION THREAD

**Discussion by 54145a** [community/community#58635] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Unable to access github in China.

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ast2008 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
